### ISSUE TITLE
npm version

### ISSUE THREAD

**Issue by aliqasemzadeh** [ionic-team/ionicons#521]
Hello,
Please add npm version.

Thanks

**Comment by Grawl** [comment]
the project uses dead Bower and dead Component but not living NPM

**Comment by dav-is** [comment]
Yeah and I'm confused why there haven't been any updates for http://ionicons.com/

**Comment by nlieb** [comment]
I cannot even compile the css manually since they removed the `builder/` folder in v4 which the `build` script in `package.json` depends on.

**Comment by christian-bromann** [comment]
This has been resolved. If there are still remaining issues, please let me know.
